### ISSUE TITLE
Linux build environment

### ISSUE THREAD

**Issue by stevemk14ebr** [ReClassNET/ReClass.NET#47]
What is the recommended build environment for compiling this on linux? A small guide would be appreciated

**Comment by KN4CK3R** [comment]
You need to install msbuild and gcc. Then you can just use the make file to build the whole project if I remember correctly.

**Comment by KN4CK3R** [comment]
Install Mono >= 5 (http://www.mono-project.com/download/stable/)
```
sudo apt-get install gcc
git clone https://github.com/ReClassNET/ReClass.NET.git
cd ReClass.NET
make update
make debug
```

Copy NativeCore.so from 
/ReClass.NET/NativeCore/Unix/build/debug
to
/ReClass.NET/ReClass.NET/bin/Debug/x64

Maybe someone with more knowlegde about make files could add the copy?

Then start ReClass.NET and get the error System.IO.FileNotFoundException: /ReClass.NET/ReClass.NET/bin/Debug/x64/NativeCore.so :confused:
Currently don't know what's the problem here.

**Comment by stevemk14ebr** [comment]
Fixed by #84 and #85
